src/mono/mono.proj

```diff
@@ -655,9 +655,34 @@
 
   <Import Project="Directory.Build.targets" />
 
+  <Target Name="BuildIosAppBuilderTasks">
+  <MSBuild Projects="$(MonoProjectRoot)IosAppBuilderTasks\IosAppBuilderTasks.csproj"
+           Properties="Configuration=$(Configuration)"
+           Targets="Restore;Build" />
+  </Target>
+  <UsingTask TaskName="IosAppBuilderTask" AssemblyFile="$(ArtifactsObjDir)mono\IosAppBuilderTasks\$(TargetArchitecture)\$(Configuration)\IosAppBuilderTasks.dll" />
+  <Target Name="BuildIosApp" DependsOnTargets="BuildIosAppBuilderTasks">
+    <IosAppBuilderTask 
+        Arch="$(TargetArchitecture)"
+        ProjectName="$(ProjectName)"
+        Optimized="$(Optimized)"
+        MonoInclude="$(BinDir)include\mono-2.0"
+        CrossCompiler="$(BinDir)cross\mono-aot-cross"
+        EntryPointLib="$(EntryPointLib)"
+        NativeMainSource="$(NativeMainSource)"
+        GenerateOnlyXcodeProject="$(GenerateOnlyXcodeProject)"
+        DevTeamProvisioning="$(DevTeamProvisioning)"
+        AppDir="$(AppDir)">
+        <Output TaskParameter="AppBundlePath" PropertyName="AppBundlePath" />
+        <Output TaskParameter="XcodeProjectPath" PropertyName="XcodeProjectPath" />
+    </IosAppBuilderTask>
+    <Message Importance="High" Text="Xcode: $(XcodeProjectPath)"/>
+    <Message Importance="High" Text="App: $(AppBundlePath)"/>
+  </Target>
+
   <!-- Ordering matters! Overwriting the Build target. -->
   <!-- General targets -->
-  <Target Name="Build" DependsOnTargets="BuildMonoRuntimeUnix;BuildMonoRuntimeWindows">
+  <Target Name="Build" DependsOnTargets="BuildMonoRuntimeUnix;BuildMonoRuntimeWindows;BuildIosAppBuilderTasks">
```

Comment:
This should be called only when building tests

Reply:
@marek-safar the iOS sample now depends on it (the project is super small so it takes less than a second to build)

Reply:
I know but it does not set good practice going forward (android version, wasm, run-tests will all just follow this). Could you add it as project reference to the sample?